deep_qa/data/datasets/__init__.py

```diff
@@ -1,7 +1,7 @@
 from collections import OrderedDict
 
-from .entailment.snli_dataset import SnliDataset
-from .language_modeling.language_modeling_dataset import LanguageModelingDataset
+from .snli_dataset import SnliDataset
+from .language_modeling_dataset import LanguageModelingDataset
```

Comment:
Oh, they're the readers.  There's no additional class - we only need one `Dataset` class, and we just have `DatasetReaders` to construct datasets with particular instances.

This was from an earlier iteration of the design, before I settled on what you see in this PR.  I'll fix this.